docs/design/visualizations/index.md

```diff
@@ -0,0 +1,92 @@
+# Visualizations
+
+CARET is a tool for performance evaluation.
+
+For evaluation, quantification and visualization would be sufficient.  
+For analysis, on the other hand, various factors are involved in behavior and performance, so it is necessary to investigate from multiple perspectives.
+
+CARET has multiple visualizations for evaluation and analysis APIs that are readily available.  
+Also, it's also possible to directly acquire and evaluate the data on your own (see [Processing trace data](../processing_trace_data/index.md) for detail).
+
+## Policy
+
+### Evaluation flow
+
+CARET has a large amount of information that can be obtained, and it is difficult to evaluate large-scale applications simply by displaying the recorded trace data in time series.
+So, it is important to focus on evaluation targets with appropriate granularity according to the purpose of evaluation.
+
+The following diagram shows the evaluation flow and analysis by adjusting the granularity.
+
+![Analysis flow](../../imgs/analysis_flow.drawio.png)
+
+Here, the horizontal axis represents the visualization granularity and the vertical axis represents the processing granularity.  
```

Comment:
Please add the meaning of visualization granularity and the processing granularity.

Reply:
I think that adding the following sentence might be kind.
```
Those definitions will be explained subsequent sections.
```